packages/ee/server/checkLicense.ts

```diff
@@ -0,0 +1,21 @@
+import cache from "memory-cache";
+
+async function checkLicense(license: string): Promise<boolean> {
+  const url = `https://cal.com/api/license?key=${license}`;
+  const cachedResponse = cache.get(url);
+  if (cachedResponse) {
+    return cachedResponse;
+  } else {
+    try {
+      const hours = 24;
+      const response = await fetch(url);
+      const data = await response.json();
+      cache.put(url, data.valid, hours * 1000 * 60 * 60);
```

Comment:
What does `hours * 1000 * 60 * 60` means? `const expireDateCacheOneMonthAhead?`

Reply:
It's 24 hours in milliseconds

Reply:
Given you also declare a var with the amount of hours, I prefer the pattern of hardcoding into a constant.

```javascript
const CACHING_TIME /* makeshift name */ = 86400000; // 24 hours in milliseconds
```